pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/OncHistoryDetail.java

```diff
@@ -62,7 +62,7 @@ public class OncHistoryDetail implements HasDdpInstanceId {
                     + "collaborator_sample_id, block_sent, scrolls_received, sk_id, sm_id, "
                     + "sent_gp, first_sm_id, additional_tissue_value_json, expected_return, return_date, return_fedex_id, "
                     + "shl_work_number, block_id_shl, tumor_percentage, tissue_sequence, "
-                    + "scrolls_count, uss_count, h_e_count, blocks_count, sm.sm_id_value, sm.sm_id_type_id, sm.sm_id_pk, sm.deleted, "
+                    + "scrolls_count, uss_count, h_e_count, blocks_count, t.deleted, sm.sm_id_value, sm.sm_id_type_id, sm.sm_id_pk, sm.deleted, "
```

Comment:
Having multiple columns in the select with the same name (`deleted`) might make for runtime errors if code is addressing columns by column name vs. index.  Maybe change these to two different aliases and update `FieldSettingsDao.DELETED`?

Reply:
I do see us doing things like `DBConstants.DDP_ONC_HISTORY_DETAIL_ALIAS + DBConstants.ALIAS_DELIMITER + DBConstants.DELETED`, which makes for some error prone result set fetching.  Maybe make some new constants like `DBConstants.DDP_ONC_HISTORY_DETAIL_DELETED` so that we don't need to remember to concat various things when looking for a column?